lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -409,6 +409,160 @@ def test_kwargs(self):
         self.assertMaskedArrayEqual(expected_result, res_cube.data)
 
 
+class Testslices(tests.IrisTest):
+    '''
+    This class tests the slices capability of iris.cube.Cube
+    '''
+    def setUp(self):
+        '''
+        setup a 4D iris cube, each dimension is length 1.
+        The dimensions are;
+            dim1: height
+            dim2: time
+            dim3: longitude
+            dim4: latitude
```

Comment:
The standard order of dimensions in Iris is T, Z, Y ,X. You could probably use one of the cubes in `iris.tests.stock`